example/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: maplibre_gl_example
 description: Demonstrates how to use the maplibre_gl plugin.
 publish_to: 'none'
-version: 1.0.0+1
+version: 1.1.0+1
```

Comment:
Thanks for the pull request. Using gradle 8 in the example app makes sense to me. 
I just wouldn't versioning the example app. Let's keep it at 1.0.0.

Reply:
Makes sense - reverted that!